variables.tf

```diff
@@ -20,6 +20,18 @@ variable "aws_sso_entity_id" {
   description = "AWS SSO Entity ID for the Okta App"
 }
 
+variable "aws_config_aggregator" {
+  type        = string
+  default     = null
+  description = "ID of the AWS account allowed to aggregate AWS Config data for all core accounts"
+}
+
+variable "aws_config_aggregator_regions" {
+  type        = list(string)
+  default     = ["eu-central-1", "eu-west-1"]
+  description = "List of regions the AWS Config Aggregator account is allowed to access"
+}
```

Comment:
Seems to me that we merge these two into an `aws_config` object variable (and drop the default IMO) 😉